.github/workflows/build_and_test.yml

```diff
@@ -248,6 +248,7 @@ jobs:
     # Run regular jobs for commit in both Apache Spark and fored repository
```

Comment:
Maybe we can add one line of more comment here such as "Run scheduled jobs with JDK 17 in Apache Spark"

Reply:
Oh, sure.